CONTRIBUTOR_LADDER.md

```diff
@@ -0,0 +1,161 @@
+
+### Contributor
+
+Description: A Contributor contributes directly to the project and adds value to it. Contributions need not be code. People at the Contributor level may be new contributors, or they may only contribute occasionally.
+
+* Responsibilities include:
+    * Follow the CNCF CoC
+    * Follow the project contributing guide
+* Requirements (one or several of the below):
+    * Sign the DCO for Open Cluster Management, either individually or through their employer
+    * Report and sometimes resolve issues
+    * Occasionally submit PRs
+    * Contribute to the documentation
+    * Show up at meetings, takes notes
+    * Answer questions from other community members
+    * Submit feedback on issues and PRs
+    * Test releases and patches and submit reviews
+    * Run or helps run events
+    * Promote the project in public
+    * Help run the project infrastructure
+* Privileges:
+    * Invitations to contributor events
+    * Eligible to become an Organization Member
+
+
+### Organization Member
+Description: An Organization Member is an established contributor who regularly participates in the project. Organization Members have privileges in both project repositories and elections, and as such are expected to act in the interests of the whole project. While Organization Members are selected by individual subprojects, some of their requirements and privileges apply to the entire OCM project.
+
+An Organization Member must meet the responsibilities and has the requirements of a Contributor, plus:
+
+* Responsibilities include:
+    * Continues to contribute regularly, as demonstrated by having at least 25 contributions a year, as demonstrated by Devstats project metrics. 
+* Requirements:
+    * Must have successful contributions to the project, including at least one of the following:
+        * 2 accepted PRs,
+        * Reviewed 3 PRs,
+        * Resolved and closed 5 Issues,
+        * Become responsible for a key project management area,
+        * Or some equivalent combination or contribution
+    * Must have been contributing for at least 3 months
+    * Must be actively contributing to at least one project area
+    * Must have two sponsors who are also Organization Members 
+      <!-- enable when we have enough multi-org contributors: at least one of whom does not work for the same employer -->
+    * Enable 2FA on their GitHub account
+
+* Privileges:
+    * May be assigned Issues and Reviews
+    * May give commands to CI/CD automation
```

Comment:
Does our CICD process allow external contributors to give commands to CICD automation?

Reply:
yes, it can by just comment on PR.